packages/common/src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -84,6 +84,7 @@ const FULL_ENDPOINT_MAP = {
   topGenreUsers: '/users/genre/top',
   topArtists: '/users/top',
   getTrack: (trackId: OpaqueID) => `/tracks/${trackId}`,
+  getTrackStreamUrl: (trackId: OpaqueID) => `/tracks/${trackId}/stream-url`,
```

Comment:
could use sdk for this instead of api client which we are trying to move away from.